apps/site-app/src/containers/Accounts/Redeem/index.tsx

```diff
@@ -9,68 +9,36 @@ import { AccountMap } from '@thecointech/shared/containers/AccountMap';
-  messageDesc : {
-      defaultMessage: 'An optional message to the recipient. Should not include the security answer',
-      description: 'app.accounts.redeem.form.messageDesc: Label for the form the make a payment page / etransfert tab'},
-  noSpecialCaractDesc : {
-      defaultMessage: 'No numbers or special characters',
-      description: 'app.accounts.redeem.form.noSpecialCaractDesc: Label for the form the make a payment page / etransfert tab'},
-  step1 : {
-      defaultMessage: 'Step 1 of 3: Checking order availability...',
-      description: 'app.accounts.redeem.step1: Message for the form the make a payment page / etransfert tab'},
-  step2 : {
-      defaultMessage: 'Step 2 of 3: Sending sell order to our servers...',
-      description: 'app.accounts.redeem.step2: Message for the form the make a payment page / etransfert tab'},
-  step3 : {
-      defaultMessage: 'Step 3 of 3: Waiting for the order to be accepted\n(check progress {link})...',
-      description: 'app.accounts.redeem.step3: Message for the form the make a payment page / etransfert tab'},
-  transferOutHeader : {
-      defaultMessage: 'Processing Transfer out...',
-      description: 'app.accounts.redeem.transferOutHeader: Message for the form the make a payment page / etransfert tab'},
-  transferOutProgress : {
-      defaultMessage: 'Please wait, we are sending your order to our servers...',
-      description: 'app.accounts.redeem.transferOutProgress: Message for the form the make a payment page / etransfert tab'},
-  button : {
-      defaultMessage: 'Send e-Transfer',
-      description: 'app.accounts.redeem.form.button: For the button in the make a payment page / etransfer tab'}
-});
-
+  step1: {
+    defaultMessage: 'Step 1 of 3: Checking order availability...',
+    description: 'app.accounts.redeem.step1: Message for the form the make a payment page / etransfert tab'
+  },
+  step2: {
+    defaultMessage: 'Step 2 of 3: Sending sell order to our servers...',
+    description: 'app.accounts.redeem.step2: Message for the form the make a payment page / etransfert tab'
+  },
+  step3: {
+    defaultMessage: 'Step 3 of 3: Waiting for the order to be accepted\n(check progress {link})...',
+    description: 'app.accounts.redeem.step3: Message for the form the make a payment page / etransfert tab'
+  },
+  transferOutProgress: {
+    defaultMessage: 'Please wait, we are sending your order to our servers...',
+    description: 'app.accounts.redeem.transferOutProgress: Message for the form the make a payment page / etransfert tab'
+  },
+})
 export const Redeem = () => {
 
   const [coinToSell, setCoinToSell] = useState(null as number | null);
-  const [email, setEmail] = useState('');
-  const [question, setQuestion] = useState('');
-  const [answer, setAnswer] = useState('');
+  const [email, setEmail] = useState<MaybeString>();
+  const [question, setQuestion] = useState<MaybeString>();
+  const [answer, setAnswer] = useState<MaybeString>();
```

Comment:
`''` is not a valid value for any of these entries.  Trying to use `undefined` for no value, rather than empty string.  It lets us do tests like `if (email)` (which prior would pass, and was causing at least one bug)